src/main/java/frc/robot/RobotContainer.java

```diff
@@ -383,6 +433,8 @@ public void periodic() {
 
       pitch.setDouble(odometry.getPitch().getDegrees());
       roll.setDouble(odometry.getRoll().getDegrees());
+
+      ledSubsystem.runLEDs();
```

Comment:
This should be in the LEDSubsystem's periodic function, not here

Reply:
If I make a periodic function in ledsubsystem, will I have to call it in the teleop periodic in robot?

Reply:
Nope! Look at `ExampleSubsystem` to see an example. The `@Override` tag tells the computer that it works with it's parent (`subsystemBase`) class, which runs multiple times a second